server/routes/reviewPlan/reviewNoteController.ts

```diff
@@ -31,6 +32,11 @@ export default class ReviewNoteController {
     const updatedReviewPlanDto = updateDtoWithFormContents(reviewPlanDto, reviewNoteForm)
     getPrisonerContext(req.session, prisonNumber).reviewPlanDto = updatedReviewPlanDto
 
+    const errors = validateReviewNote(reviewNoteForm)
```

Comment:
Is it worth covering this new code branch with a unit test?
Have a look at `it('should redisplay page given form submitted with validation errors', async () => {` in `whoCompletedReviewController.test.ts` for a similar example